packages/flutter/test/material/slider_test.dart

```diff
@@ -186,138 +189,209 @@ void main() {
+
+    // Test default theme for disabled widget.
+    await tester.pumpWidget(buildApp(enabled: false));
+    await tester.pump(const Duration(seconds: 1)); // wait for disable animation to finish.
+    expect(
+        sliderBox,
+        paints
+          ..rect(color: sliderTheme.disabledActiveRailColor)
+          ..rect(color: sliderTheme.disabledInactiveRailColor));
+    expect(sliderBox, paints..circle(color: sliderTheme.disabledThumbColor));
+    expect(sliderBox, isNot(paints..circle(color: sliderTheme.thumbColor)));
+    expect(sliderBox, isNot(paints..rect(color: sliderTheme.activeRailColor)));
+    expect(sliderBox, isNot(paints..rect(color: sliderTheme.inactiveRailColor)));
+
+    // Test setting the activeColor and inactiveColor for disabled widget.
+    await tester.pumpWidget(
+        buildApp(activeColor: customColor1, inactiveColor: customColor2, enabled: false));
+    expect(
+        sliderBox,
+        paints
+          ..rect(color: sliderTheme.disabledActiveRailColor)
+          ..rect(color: sliderTheme.disabledInactiveRailColor));
+    expect(sliderBox, paints..circle(color: sliderTheme.disabledThumbColor));
+    expect(sliderBox, isNot(paints..circle(color: sliderTheme.thumbColor)));
+    expect(sliderBox, isNot(paints..rect(color: sliderTheme.activeRailColor)));
+    expect(sliderBox, isNot(paints..rect(color: sliderTheme.inactiveRailColor)));
+
+    // Test that the default value indicator has the right colors.
+    await tester.pumpWidget(buildApp(divisions: 3));
+    Offset center = tester.getCenter(find.byType(Slider));
+    TestGesture gesture = await tester.startGesture(center);
+    await tester.pump();
+    await tester
+        .pump(const Duration(milliseconds: 500)); // wait for value indicator animation to finish.
+    expect(value, equals(2.0 / 3.0));
+    expect(
+      sliderBox,
+      paints
+        ..rect(color: sliderTheme.activeRailColor)
+        ..rect(color: sliderTheme.inactiveRailColor)
+        ..circle(color: sliderTheme.overlayColor)
+        ..circle(color: sliderTheme.activeTickMarkColor)
+        ..circle(color: sliderTheme.activeTickMarkColor)
+        ..circle(color: sliderTheme.inactiveTickMarkColor)
+        ..circle(color: sliderTheme.inactiveTickMarkColor)
+        ..path(color: sliderTheme.valueIndicatorColor)
+        ..circle(color: sliderTheme.thumbColor),
+    );
+    await gesture.up();
+    await tester.pump();
+    await tester
+        .pump(const Duration(milliseconds: 500)); // wait for value indicator animation to finish.
+
+    // Testing the custom colors are used for the indicator.
+    await tester.pumpWidget(buildApp(
+      divisions: 3,
+      activeColor: customColor1,
+      inactiveColor: customColor2,
+    ));
+    center = tester.getCenter(find.byType(Slider));
+    gesture = await tester.startGesture(center);
+    await tester.pump();
+    await tester
+        .pump(const Duration(milliseconds: 500)); // wait for value indicator animation to finish.
```

Comment:
several times in this test the comment caused dartfmt to wrap a "pump" call inconsistently with the rest of the code